pkg/machines/libvirt-dbus.js

```diff
@@ -1338,6 +1339,14 @@ export function attachIface({ connectionName, vmId, mac, permanent, hotplug, sou
     return attachDevice({ connectionName, vmId, permanent, hotplug, xmlDesc });
 }
 
+export function changeDiskSettings(connectionName, objPath, target, readonly, shareable) {
```

Comment:
Maybe better changeDiskAccessPolicy is better, but I don't really mind this name either.